webdriver/webdriverio/test/specs/example.e2e.js

```diff
@@ -0,0 +1,32 @@
+// calculates the luma from a hex color `#abcdef`
```

Comment:
This spec seems to be very generic and it will fail for our application. Please replace it with a useful first test for note.rs, e.g. you may test for the presence of title bar buttons are present or the transition from main to settings screen.